### ISSUE TITLE
Cyst of breast

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10079]
**Preferred term label:**
Cyst of breast

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0006144

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Galactocele

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001046
- label: Breast cyst
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: 
- def: A benign, fluid-filled sac in the breast.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10079
- synonyms:
